multi-progress.js

```diff
@@ -19,82 +21,69 @@ const mockInstance = {
   isTTY: false,
 };
 
-module.exports = (Progress) => {
-  class MultiProgress {
-    constructor(stream) {
-      this.stream = stream || process.stderr;
-      this.isTTY = this.stream.isTTY;
-
```

Comment:
Do you mind adding back the blank lines visible in the diff here?

https://github.com/pitaj/multi-progress/pull/12/files?w=1

Reply:
Done. My editor removed them automatically